src/core/getters/array.ts

```diff
@@ -24,7 +24,7 @@ export const getArray = ({
       context,
     });
     return {
-      value: `${resolvedObject.value}[]`,
+      value: `(${resolvedObject.value})[]`,
```

Comment:
We could maybe add a check if the string contain a |. Like that we can just add the parentheses when needed.

Reply:
imo its not worth adding more complexity here. can use the `--prettier` if you really care about clean generated code